includes/apple-exporter/components/class-podcast.php

```diff
@@ -94,7 +94,7 @@ private static function get_podcast_url( $node ) {
 		// Pattern match src attribute for apple podcast url.
 		$url = $iframe->getAttribute( 'src' );
 
-		if ( empty( $url ) || ! str_contains( $url, 'podcasts.apple.com' ) ) {
+		if ( empty( $url ) || false === strpos( $url, 'podcasts.apple.com' ) ) {
```

Comment:
Sneaking this fix in here.  This is the php < 8 compatibility issue Grant spotted.  `strpos` should work on php 7.x versions.

Reply:
Good call, thanks